apps/backend/src/libs/plugins/authorization-plugin/libs/helpers/check-white-route.helper.ts

```diff
@@ -0,0 +1,9 @@
+const checkWhiteRoute = (route: string, whiteRoutes: string[]): boolean => {
```

Comment:
```suggestion
const checkIsWhiteRoute = (route: string, whiteRoutes: string[]): boolean => {
```

Reply:
Do I need to rename the file into `check-is-white-route.helper.ts`?

Reply:
Yes